components/automate-ui/src/app/pages/job-add/job-add.component.ts

```diff
@@ -98,29 +98,27 @@ export class JobAddComponent implements OnDestroy {
     }, {
       validator: profileSelectionRequiredValidator
     });
-    const defaultStart = moment();
-    const defaultEnd = moment(defaultStart).add(1, 'days');
+    const defaultStart = moment.utc();
+    const defaultEnd = moment.utc(defaultStart).add(1, 'days');
     const scheduleGroup = this.fb.group({
       name: ['', Validators.required],
       include: false,
       start: this.fb.group({
         datetime: this.fb.group({
-          month: defaultStart.month() + 1,
+          month: defaultStart.month(),
           date: defaultStart.date(),
           year: defaultStart.year(),
-          hour: this.hours24to12(defaultStart.hour()),
-          minute: defaultStart.minute(),
-          meridiem: defaultStart.hour() - 12 > 0 ? 'PM' : 'AM'
+          hour: defaultStart.hour(),
+          minute: defaultStart.minute()
         })
```

Comment:
No longer using AM/PM - so we've removed it from the FormGroup.  We also no longer need to convert the hours from 12h to 24h, so we can remove that function because UTC formate is 24h